pkgs/top-level/python-packages.nix

```diff
@@ -2964,6 +2964,33 @@ in {
     };
   };
 
+  httpserver = buildPythonPackage rec {
```

Comment:
please move expression to `pkgs/development/python-modules/httpserver/default.nix`